example_simple_classification.py

```diff
@@ -108,14 +109,18 @@ def main():
     # Save results to db
     # TODO Save results to db
 
-
     # Look at the RF feature importance rankings
     # TODO this is broken
     # hcai.get_advanced_features().plot_rffeature_importance(save=False)
 
     # Create ROC plot to compare the two models
     # TODO this is broken - it might look like tools.plot_roc(models=[random_forest, linear, knn])
-    # hcai.plot_roc()
+
+    # Create a single ROC plot from the trained model
+    trained_model.roc_curve_plot()
```

Comment:
Oh, that's a great idea!
I'll have to think about where to implement that. Noted in #239